.github/workflows/gh-pages.yml

```
@@ -18,8 +18,8 @@ jobs:
       uses: input-output-hk/setup-haskell@v1
       id: setup-haskell
       with:
-        ghc-version: "9.2.8"
-        cabal-version: "3.12"
+        ghc-version: "9.6.7"
+        cabal-version: "3.14"
```

I think the `gh-pages` workflow will start failing if you merge this line

---

We can check this by running the workflow manually with this branch. IIRC, it won't update the site if the branch isn't `master`.

---

If it does, we'll have to fix it, since we are dropping ghc-9.2 support.

---

Or is there a problem with cabal-3.14?

---

Yes, I was referring to the cabal-version change. (I thought GitHub would make it clear which line I was referring to when I made the comment!)

---

See #4840

---

That's fine. We can use an older version of cabal for haddock until either problem resolves itself with newer version of cabal or we will be forced to find a solution. Whichever comes first.
```suggestion
        cabal-version: "3.12"
```